<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addtocart_dialog</class>
 <widget class="QDialog" name="addtocart_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>593</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="discardBotton">
   <property name="geometry">
    <rect>
     <x>459</x>
     <y>160</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Discard</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="product_table">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>511</width>
     <height>61</height>
    </rect>
   </property>
   <row>
    <property name="text">
     <string/>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Product</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Manufacturer</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Price</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Expiry Date</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Amount:</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="amountSpinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>10000</number>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
